packages/reakit/src/Grid/index.ts

```diff
@@ -1,6 +1,7 @@
 import Grid from "./Grid";
 import GridItem from "./GridItem";
 
+// @ts-ignore
 Grid.Item = GridItem;
```

Comment:
Take a look at how we did it in `Card`:
https://github.com/reakit/reakit/blob/50a525ebf4f2ff20aa4238413c6b9af7fd6cfcdc/packages/reakit/src/Card/index.ts#L1-L14

Also, we have to `export * from Grid` and from `GridItem` so the props interfaces will be available for the users.